src/locales/us/en/index.json

```diff
@@ -75,6 +75,7 @@
   "shouldContainALowercase": "Should contain a lowercase letter",
   "shouldContainASymbol": "Should contain a symbol",
   "showPassword": "Show password",
+  "signalsEvaluation": "Evaluating...",
```

Comment:
Could be as simple as "Checking device..."
or slightly more specific "Checking your device..." maybe?
Otherwise in their docs I found a sentence we could lift; "Calculating risk level...". Might be too techy though?